app/models/feature_toggler.rb

```diff
@@ -0,0 +1,12 @@
+class FeatureToggler
+  def initialize(feature_flags)
+    @feature_flags = feature_flags
+  end
+
+  def use_recommended_related_links?(content_item_links, request_headers)
+    return false if content_item_links.nil?
+
+    (!content_item_links.has_key?('ordered_related_items') || content_item_links['ordered_related_items'].empty?) &&
```

Comment:
I think `content_item_links.fetch('ordered_related_items', []).empty?` would be clearer

Reply:
Good shout.